package.json

```diff
@@ -25,7 +25,7 @@
     "@alcalzone/release-script-plugin-license": "^3.7.0",
     "@alcalzone/release-script-plugin-manual-review": "^3.7.0",
     "@iobroker/testing": "^4.1.3",
-    "@tsconfig/node18": "^18.2.4",
+    "@tsconfig/node20": "^20.18.0",
```

Comment:
Ok, Schnellschuss hat nicht funktioniert.  Ich hab ja sowas von keine Ahnung von ioBroker und node, aber das wird schon... ;-)

Warum haben wir node in den devDependencies, aber nicht oben in den dependencies?!